apps/client/src/styles.scss

```diff
@@ -1 +1,13 @@
-/* You can add global styles to this file, and also import other style files */
+@tailwind components;
+@tailwind base;
+@tailwind utilities;
+
+@import url('https://fonts.googleapis.com/css2?family=Major+Mono+Display&family=Roboto+Mono:wght@400;500&family=Sora:wght@400;600&display=swap');
+
+/* font-family: 'Major Mono Display', monospace;
```

Comment:
Do we need the commented code here?

Reply:
These comments are removed in PR #43 under commit https://github.com/the-ai-team/razor/pull/43/commits/31bd36e8f931f9879a23fc1eeca7223db3390113